### DISCUSSION TITLE
Direct sharing to iOS app

### DISCUSSION THREAD

**Discussion by neldonado** [karakeep-app/karakeep#325] [Ideas]
I think a cool feature would be to enable direct sharing via iOS share menu. Being able to long press a photo and just automatically add it to the hoarder app would be really seamless.

![image](https://github.com/user-attachments/assets/f5a4b9e0-66b9-4783-8f27-d56d7e97c52e)

**Comment by MohamedBassem** [comment]
Hey, Unless I'm missing something, but Hoarder exists in iOS' share menu already.

![image](https://github.com/user-attachments/assets/011437c4-52c0-43f8-8c61-2a1de7d819f8)

**Reply by neldonado** [reply]
Alright I swear I triple checked everything and didn't see it. I went back just now and it's there. Maybe there was a little lag time between installing the app and it showing up? Or maybe I was just blind 🤪 thank you!! Amazing app!
